li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerBuilder.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").  See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.clients.consumer;
+
+import java.util.Map;
+
+
+public class LiKafkaConsumerBuilder<K, V> {
```

Comment:
the default constructor uses a null map, which means new Builder().build() will get me an IllegalStateException. i think its more usable to default to an empty map.

Reply:
Done